From e92cfbf246df1b0cd48e17de75edcb724efe5620 Mon Sep 17 00:00:00 2001
From: Ilya Maximets <i.maximets@ovn.org>
Date: Thu, 26 Sep 2019 12:03:09 +0300
Subject: travis: Drop -MD related workaround for sparse.

The issue was fixed in upstream sparse by the following commit:
d90c0838c101 ("cgcc: fix wrong processing of -MD & -MMD")

This patch is required to fix our travis build.

Signed-off-by: Ilya Maximets <i.maximets@ovn.org>
Acked-by: Ian Stokes <ian.stokes@intel.com>
---
 .travis/linux-prepare.sh | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/.travis/linux-prepare.sh b/.travis/linux-prepare.sh
index 65348c9f9..70fd98f71 100755
--- a/.travis/linux-prepare.sh
+++ b/.travis/linux-prepare.sh
@@ -9,11 +9,6 @@ set -ev
 # linking against it fails.
 git clone git://git.kernel.org/pub/scm/devel/sparse/sparse.git
 cd sparse
-# Commit bb1bf748580d ("cgcc: gendeps for -MM, -MD & -MMD too") makes
-# sparse ignore almost all source files, because 'make' uses '-MD' to
-# generate dependencies as a side effect within compilation commands.
-git revert bb1bf748580d --no-commit
-git diff HEAD
 make -j4 HAVE_LLVM= install
 cd ..
 
-- 
2.14.1


